src/bp-core/js/mentions.js

```diff
@@ -380,19 +380,7 @@ window.bp = window.bp || {};
 		var opts = $.extend(true, {}, suggestionsDefaults, mentionsDefaults, options);
 		return $.fn.atwho.call(this, opts);
 	};
-	
-	$(document).ready(function () {
```

Comment:
@kishanjasani Can you check comment history for this before removing code? Might be this code added to fix some issue as I know.

Reply:
Not sure what this code is doing. But it's updating dom on the key events. Which is not an appropriate solution. If this code is really required then we need to find some better solution for it.

Reply:
@kishanjasani I'm not sure. this code is required or not. I think this code is here for some reason so we need to think twice before removing the code. 

@chetansatasiya @KartikSuthar Can you look into it.

Reply:
@kishanjasani Please check your removed code will be fixed all the issues mentioned in the PR:
https://github.com/buddyboss/buddyboss-platform/pull/2042